### DISCUSSION TITLE
如何可以通过HA自动化给小爱音箱发送指令

### DISCUSSION THREAD

**Discussion by pigstrong9** [XiaoMi/ha_xiaomi_home#51] [General]
目前有一个需求，美的的洗衣机洗完衣服给小爱音箱指令，小爱播放消息，但是HA里没看到有这个事件，有一个通知，不知道怎么用

**Comment by FDrag0n** [comment]
在自动化或者脚本那里给小爱音箱发通知事件就行，有执行命令实体和播放消息两个实体，按需选择即可

**Reply by pigstrong9** [reply]
大佬能否截个图，不知道选哪个

**Reply by FDrag0n** [reply]
> 大佬能否截个图，不知道选哪个

![image](https://github.com/user-attachments/assets/c0ab0c9b-a5c7-48ca-8534-2779ddd703f7)
要发送的消息用冒号和中括号框起来就行

**Reply by zwlhome** [reply]
> 在自动化或者脚本那里给小爱音箱发通知事件就行，有执行命令实体和播放消息两个实体，按需选择即可

请问大佬执行指令怎么操作呢，同样的格式报错了

**Reply by FDrag0n** [reply]
我还没试执行指令，应该差不多吧，晚上回去试下

**Reply by zwlhome** [reply]
> 我还没试执行指令，应该差不多吧，晚上回去试下

解决了，["xxx",0]。0代表响应，1代表静默

**Reply by FDrag0n** [reply]
> > 我还没试执行指令，应该差不多吧，晚上回去试下
> 
> 解决了，["xxx",0]。0代表响应，1代表静默

可以的，学到了，晚上回去试试
